history.py

```diff
@@ -0,0 +1,26 @@
+from catalog import models as cmod
+
+class LastFiveMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+        # One-time configuration and initialization.
```

Comment:
Comments are better to be used before the codes. Sometimes it is confusing to read when comments are below the codes.

Reply:
Brian, this was the default code from djangoproject.com